homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -121,7 +124,13 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
         _LOGGER.debug("Initializing with host %s (token %s...)", host, token[:5])
         vacuum = Vacuum(host, token)
```

Comment:
We should probably not create a new device here but use the same device as the coordinator is using.